lib/cdo/aws/ec2_autoscaling.rb

```diff
@@ -23,7 +23,8 @@ def self.refresh_instances_in_group(group_name, wait: false)
         preferences: {
           min_healthy_percentage: 100,
           max_healthy_percentage: 200,
-          instance_warmup: 0
+          instance_warmup: 0,
+          skip_matching: false # replace instances even if there are no launch template changes
```

Comment:
I assume this is maintaining a default based on gem change rather than a change in behavior?

Reply:
No behavior change here, `skip_matching` defaults to false - I decided to specify it because it is part of the requirements driving this change. The actual reason we want the ability to programatically refresh instances is not because the AMI or launch template has changed, but because we want to refresh caches and simply replacing instances is far easier and reliable than trying to fan out commands to all running instances.